app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,19 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>BudgetApp</title>
+    <title>EveryCent</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
   </head>
 
   <body>
-    <%= yield %>
+    <div class="main-app">
+      <%= yield %>
```

Comment:
Great job @iAmWaris97 🤝

- [ ] [Optional] I would suggest you add a back button or logout button to this page because I can't seem to navigate back to the previous pages.
![no-back](https://user-images.githubusercontent.com/63917725/221391709-1be683bc-b608-4282-9e7d-eb09fc5fa77b.PNG)

Reply:
Thank you for the kind review!

This is the root page once a user logs in!

However, I will add a sign_out path to the header so that a user can sign out.